app/javascript/packs/checkbox.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function(){
```

Comment:
make all this functions arrow functions, and rename this module as seems that it used for product deleting